### PR TITLE
Create adp.yml (ADP)

### PR DESCRIPTION
**Pull request by jcwoods**
Adds or updates the following companies, events, or universities:
 - 
 - 

#### Company updates
 - [x] I have put the most recent relevant date in the "Last Update" column
 - [ ] I have linked to the article about the change, not the company's homepage (Please put N/A if there is no public post)

#### Event updates
 - [ ] I have linked to the article about the change, not the event's homepage

#### University updates
 - [ ] I have linked to the article about the change, not the university's homepage

### CODE REVIEW

File: _data/companies/adp.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: ADP
```
Comment (drewbrew) [inline _data/companies/adp.yml]: Is this the document/payroll processing company or someone else entirely?

**Comment by jcwoods** [comment]
Yes, the HR/payroll company.

File: _data/companies/adp.yml (hunk 1 above)
Comment (jcwoods) [reply _data/companies/adp.yml]: Yes, the HR/payroll company.

Comment (drewbrew) [reply _data/companies/adp.yml]: Cool, thanks!

Comment (jcwoods) [reply _data/companies/adp.yml]: Notifications have been sent internally -- I'm not able to provide a link to external policy page.

**Pull request merged**